redisCache.py

```
@@ -106,4 +122,70 @@ async def redis_subscribe(redis: Redis, channel: str):
+        print("\033[91mERROR-DB:\033[0m Streams dictionary is empty.")
+        return None
+
+    try:
+        # Build the XREAD command as a list
+        command = ["XREAD"]
+        if count is not None:
+            command.extend(["COUNT", str(count)])
+        if block is not None:
+            command.extend(["BLOCK", str(block)])
+        command.append("STREAMS")
+        for channel, last_id in streams.items():
```

Ruff (0.13.1)

Loop control variable `last_id` not used within loop body

(B007)